packages/throttle/README.md

```diff
@@ -67,11 +74,18 @@ const useMyCallback = (initialState, wait, leading) => {
 
 #### Options
 
-| Property | Type       | Default | Description                                                                                                                |
-| -------- | ---------- | ------- | -------------------------------------------------------------------------------------------------------------------------- |
-| callback | `Function` |         | This is the callback you want to throttle                                                                                  |
-| fps      | `number`   | `30`    | Defines the rate in frames per second with which `setState` is invoked with new state                                      |
-| leading  | `boolean`  | `false` | Calls `setState` on the leading edge (right away). When `false`, `setState` will not be called until the next frame is due |
+| Property        | Type              | Default | Description                               |
+| --------------- | ----------------- | ------- | ----------------------------------------- |
+| callback        | `Function`        |         | This is the callback you want to throttle |
+| throttleOptions | `ThrottleOptions` |         | Defines throttling options                |
+
+##### ThrottleOptions
+
+| Key     | Type      | Default | Description                                                                                                                |
+| ------- | --------- | ------- | -------------------------------------------------------------------------------------------------------------------------- |
+| fps     | `number`  | `30`    | Defines the rate in frames per second with which `setState` is invoked with new state.                                     |
+| wait    | `number`  |         |                                                                                                                            |
```

Comment:
I didn't find a good way and words how to explain that only one of `fps` or `wait` options can be used.
@jaredLunde maybe you have any idea?